### ISSUE TITLE
Add term toeing out

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10499]
**Preferred term label:**
Toeing out

**Synonyms**
Duck feet, external rotated feet, Out-toeing

**Definition (free text, please give PubMed ID)**
The feet turn excessively outward when walking, caused by a rotational variation in the leg. PMID: 36098609

**Parent term (use hpo.jax.org/app)**
Gait disturbance HP:0001288

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, diaphyseal dysplasia (OMIM131300), PMID15326622

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001056
- label: Out-toeing
- parents: [Gait disturbance (HP:0001288)](https://hpo.jax.org/app/browse/term/HP:0001288)
- pmid: PMID:36098609
- def: The feet turn excessively outward when walking, caused by a rotational variation in the leg.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10499
- synonyms: External rotated feet(NONE)|Duck feet(NONE)|Toeing out(NONE)
